### PR TITLE
terminate node workspace in test tearDown functions

### PR DESCRIPTION
**Pull request by WinnieXU1**

### CODE REVIEW

File: test/functional/test_cli_commands.py [hunk 1]
```diff
@@ -463,6 +463,12 @@ def tearDown(self):
         node1.stop()
         node1.terminate_workspace()
         self.node_info = None
+        fake_config_2 = fixtures.FakeConfig()
+        node_info_2 = fake_config_2.get_node_info()
```
Comment (turtle-fly) [inline test/functional/test_cli_commands.py]: @WinnieXU1 the node might be left is named `test2`, to destroy this node, you need `node_info_2['name']="test2"` here

File: test/functional/test_node_ports.py [hunk 2]
```diff
@@ -245,6 +251,7 @@ def test_start_node_with_conflict_bmc_connection_port(self):
             assert True
         else:
             assert False
+        node2.terminate_workspace()
```
Comment (turtle-fly) [inline test/functional/test_node_ports.py]: @WinnieXU1 you write a tearDown, what are these terminates for in each test?

File: test/functional/test_cli_commands.py (hunk 1 above)
Comment (WinnieXU1) [inline test/functional/test_cli_commands.py]: @turtle-fly  I have changed the code following your suggestions. 😃

**Comment by turtle-fly** [comment]
:two_men_holding_hands: find one more YES

**Comment by fub2** [comment]
+1

**Pull request merged**